lib/dom-parser.js

```diff
@@ -247,10 +253,7 @@ function appendElement (hander,node) {
 }//appendChild and setAttributeNS are preformance key
 
 //if(typeof require == 'function'){
-var sax = require('./sax');
-var XMLReader = sax.XMLReader;
-var ParseError = sax.ParseError;
-var DOMImplementation = exports.DOMImplementation = require('./dom').DOMImplementation;
+exports.DOMImplementation = require('./dom').DOMImplementation;
```

Comment:
nit: in alignment with the previous suggestions:
```suggestion
/**
 * @deprecated Import/require from main entry point instead
 */
exports.DOMImplementation = dom.DOMImplementation;
```